src/component/TodoFilter.js

```diff
@@ -0,0 +1,34 @@
+import { findFilterByClassList } from "../utils/FILTER.js";
+import $store from "../store/index.js";
+
+export default function TodoFilter({ filterTodo }) {
+  const $filter = document.querySelector(".filters");
+
+  const init = () => {
+    const targetClass = "." + $store.todo.getFilter();
+    const target = document.querySelector(targetClass);
+    target.classList.add("selected");
+  };
+
+  const changeSelected = (target) => {
+    const $previous = $filter.querySelector(".selected");
+    $previous.classList.remove("selected");
+    target.classList.add("selected");
```

Comment:
```suggestion
    const $previous = $filter.querySelector(".selected");
    $previous.classList.toggle('selected')
```

```classList.toggle()```메서드를 사용하면 보다 간결하게 작성 가능할거 같습니다!

Reply:
해당 부분은 toggle과는 성격이 다른 로직으로 보입니다.
toggle이라 함은 있을 경우 지우고 없을 경우 만드는 것, 분기처리까지 포함한 것이 주 목적인데,
해당 로직은 분기의 성격이 없습니다.
이전에 선택되었던 필터에 **확실하게 있는** 클래스를 지우고, 새로 선택되어야하는 필터에 **확실하게 없는** 클래스를 추가하는 것이니까요.

물론 일부 취향차이와 견해 차이입니다만...🤔

Reply:
그렇군요! hwookim님이 말씀주신 것처럼 **확실하게 있는** 클래스를 지우고 **확실하게 없는**클래스를 추가한다는 관점에서 보면 제가 코멘트 남긴 코드보다 훨씬 좋아보이네요!! 감사합니다!!👍👍👍